### ISSUE TITLE
Process Websocket messaging 

### ISSUE THREAD

**Issue by 0xdead4ead** [0xdead4ead/BeastHttp#16]
Make it possible to create Websocket (plain/SSL) session. (Other repository!)

**Comment by moneroexamples** [comment]
Just wonder what is the status of the websocket support? Your `Other repository` is deprecated from what I understand?

**Comment by 0xdead4ead** [comment]
The current repository with websocket proto. will be rewritten again! Now I do not have much time to do it. I already have a part of working code. If this is important to you, i can finish it soon.

**Comment by moneroexamples** [comment]
Thanks for status update. Its all good. No reason to rush. Take your time.

**Comment by moneroexamples** [comment]
Related question. What about support for server-side events?

**Comment by 0xdead4ead** [comment]
yes. To send events there will be a other `push` member and `wait` for waiting by timer.

**Comment by moneroexamples** [comment]
Cool. This will be added to this repository or websocket's one?

**Comment by 0xdead4ead** [comment]
To this repository!

**Comment by saurik** [comment]
Can I maybe commit some funds towards getting websockets support integrated into this framework?

**Comment by 0xdead4ead** [comment]
The websocket extension will be ready soon! Currently, I am working on another project and have not thought about additional funding, I am writing code mostly in my free time. In the future, if required, I will add an opportunity for donation.

**Comment by saurik** [comment]
(Has there been any progress on this?)

**Comment by 0xdead4ead** [comment]
Hello! First of all, I am grateful to you for your interest in this project. Unfortunately, it has some fails. And there is extra code in it. First me need to fix it. Now I have another idea, which I gradually work on as far as possible. This project was originally encoded by me as part of the main program and I could not know what they would be interested in in principle. Since I have not wasted all this time, I learned a lot of things and got enough experience. Therefore, since it happened, I promise that I will bring this project to normal. 👌 As for websocket, I still have the modified code (do not confuse with what is now uploaded to BeastWebsocket repository), but I don't know working it is. Excuse me if you expected this.😢
